tests/suites/test_suite_ecp.data

```diff
@@ -210,60 +210,60 @@ depends_on:MBEDTLS_ECP_DP_CURVE448_ENABLED
 ecp_write_binary:MBEDTLS_ECP_DP_CURVE448:"3eb7a829b0cd20f5bcfc0b599b6feccf6da4627107bdb0d4f345b43027d8b972fc3e34fb4232a13ca706dcb57aec3dae07bdc1c67bf33609":"0":"1":MBEDTLS_ECP_PF_COMPRESSED:"0936f37bc6c1bd07ae3dec7ab5dc06a73ca13242fb343efc72b9d82730b445f3d4b0bd077162a46dcfec6f9b590bfcbcf520cdb029a8b73e":55:MBEDTLS_ERR_ECP_BUFFER_TOO_SMALL
 
 ECP read binary #1 (zero, invalid ilen)
-depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
-ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"0000":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
+depends_on:MBEDTLS_ECP_DP_SECP256R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP256R1:"0000":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
 
 ECP read binary #2 (zero, invalid first byte)
-depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
-ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"01":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
+depends_on:MBEDTLS_ECP_DP_SECP256R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP256R1:"01":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
 
 ECP read binary #3 (zero, OK)
-depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
-ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"00":"01":"01":"00":0
+depends_on:MBEDTLS_ECP_DP_SECP256R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP256R1:"00":"01":"01":"00":0
 
 ECP read binary #4 (non-zero, invalid ilen, too short)
-depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
-ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"04001122":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
+depends_on:MBEDTLS_ECP_DP_SECP256R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP256R1:"04001122":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
 
 ECP read binary #4a (non-zero, invalid ilen, too short)
-depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
-ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"03001122":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
+depends_on:MBEDTLS_ECP_DP_SECP256R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP256R1:"03001122":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
 
 ECP read binary #4b (non-zero, invalid ilen, too short)
-depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
-ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"02001122":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
+depends_on:MBEDTLS_ECP_DP_SECP256R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP256R1:"02001122":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
 
 ECP read binary #4c (non-zero, invalid ilen, too long)
-depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
-ecp_read_binary:MBEDTLS_ECP_DP_SECP192R1:"040011223344556677889900112233445566778899001122334455":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
+depends_on:MBEDTLS_ECP_DP_SECP256R1_ENABLED
+ecp_read_binary:MBEDTLS_ECP_DP_SECP256R1:"0400112233445566778899001122334455667788990011223344556677889900112233":"01":"01":"00":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
```

Comment:
Probably pre-existing, but it seems to me this is not too long for uncompressed format: compare to the length of the input in "#6 (non-zero, OK)" below.

I'd also like to make the titles more explicit: "non-zero, uncompress, ilen too long" for this one, (starting with 04) then "non-zero, compressed, parity 0, ilen too long" for the next one, same with parity 1 for the next.